ghost/i18n/locales/sk/comments.json

```diff
@@ -10,42 +10,42 @@
     "Add reply": "Pridať odpoveď",
     "Already a member?": "Už ste členom?",
     "Anonymous": "Anonymne",
```

Comment:
this should be a label for an anonymous user.  Is Anonym correct?  (Noun, not adverb)

Reply:
If it is a label so then yes it should be "Anonym"